build-in-container/mariner-docker-builder.sh

```diff
@@ -76,8 +103,8 @@ while (( "$#")); do
     -i ) container_type="interactive"; shift ;;
     -c ) cleanup; exit 0 ;;
     --mariner_dir ) mariner_dir="$(realpath $2)"; shift 2 ;;
-    --RPM_repo_file ) enable_custom_repofile=true; cp $(realpath $2) $custom_repo_file; shift 2 ;;
-    --RPM_container_URL ) enable_custom_repo_storage=true; RPM_container_URL="$2"; shift 2 ;;
+    --RPM_repo_file ) enable_custom_repofile=true; validate_custom_repo_file "$2"; copy_custom_repo_file "$2"; shift 2 ;;
+    --RPM_container_URL ) enable_custom_repo_storage=true; RPM_container_URL=$(echo $2 | tr " " ","); shift 2 ;;
```

Comment:
Can we just keep " " as our delimiter? It looks like the tool that consumes this just runs `tr` again to replace the "," with whitespace anyways.